index.html

```diff
@@ -0,0 +1,160 @@
+            </ul>
+            <p class="works_parragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="languages">
+                <li class="lang_skill">html</li>
+                <li class="lang_skill">css</li>
+                <li class="lang_skill">javaScript</li>
+            </ul>
+            <a href="#" class="button">See Project</a>
+        </div>   
+    </section>
+    <section class="about_myself">
+        <h3 class="about_title">About<br>Myself</h3>
+        <p class="about_parragraph">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+        <a href="#" class="final_msj2">LET'S CONNECT</a>
+        <ul class="about_list">
+            <li><a href="#" class="about_icon"><img src="./image/github_icon.png" alt="github"></a></li>
+            <li><a href="#" class="about_icon"><img src="./image/Linkedin icon.png" alt="linkedin_icon"></a></li>
+            <li><a href="#" class="about_icon"><img src="./image/V_icon.png" alt="V_icon"></a></li>
+            <li><a href="#" class="about_icon"><img src="./image/twitter_icon.png" alt="twitter_icon"></a></li>
+            <li><a href="#" class="about_icon"><img src="./image/m_icon.png" alt="m_icon"></a></li>
+        </ul>
+
+        <a href="#" class="button2">Get my resume</a>
+
+        <ul class="about_group">
+            <li>
+                <div class="lang_tech">
+                    <h3 class="lang_title">Languages</h3>
+                    <img class="lang_arrow" src="./image/Disabled.png" alt="">
+                </div>
+                
+                <ul class="about_flex">
+                    <li class="about_skill">
+                        <img class="about_img" src="./image/JS_icon.png" alt="">
+                        <h4 class="about_lang">JavaScript</h4>
+                    </li>
+    
+                    <li class="about_skill">
+                        <img class="about_img" src="./image/html_icon.png" alt="">
+                        <h4 class="about_lang">HTML</h4>
+                    </li>
+    
+                    <li class="about_skill">
+                        <img class="about_img" src="./image/css_icon.png" alt="">
+                        <h4 class="about_lang">CSS</h4>
+                    </li>
+    
+                    
+                </ul>
+            </li>
+            
+
+            <li class="lang_tech border">
+                <h3 class="lang_title">Frameworks</h3>
+                <img class="lang_arrow" src="./image/narrow.png" alt="">
+            </li>
+
+            <li class="lang_tech border">
+                <h3 class="lang_title">Skills</h3>
+                <img class="lang_arrow" src="./image/narrow.png" alt="">
+            </li>
+
+
+        </ul>
```

Comment:
I think there should be one space line and it's looks beautiful also professional.  :100: